### ISSUE TITLE
Fetches all tags every time we click the images tab

### ISSUE THREAD

**Issue by jonassode** [atc-/docker-registry-ui#147]
We have a huge repo with dozens of repos and 100 of tags. 

It's unusable for us if all tags are fetched every time the images tab are clicked

It should be a images tab which just lists the images ( sans tags ) and when selecting an image it should be

**Comment by atc-** [comment]
Agreed, I've been meaning to look at this previously. Perhaps retrieving tags beyond `:latest` only on the "further details" page.

**Comment by jonassode** [comment]
That would work much better for us. 

It's not optimal as some of our images do not have a :latest ( The are built by the CI system and deployed automatically with tag called the same as the commit or a timestamp, so latests does not really make sense. ) But that solution would still be better, currently it is not usable for us as loading the images tab takes several minutes. ( I checked and we actually have thousands of tags, not hundres )

**Comment by jonassode** [comment]
If you do an empty search you a list of the repositories, that is what I would have expected to have on the page, and then when you click a repository you get the list of available tags.

**Comment by atc-** [comment]
Yes, that's exactly what I mean - that way we'd get a much more responsive repository list page.
